code/modules/mob/living/carbon/human/species_types/voxes.dm

```diff
@@ -0,0 +1,12 @@
+/datum/species/vox
+	name = "Vox Person"
+	id = "vox"
+	say_mod = "screes"
+	sexes = FALSE
+	species_traits = list(SPECIES_ORGANIC)
+	mutanttongue = /obj/item/organ/tongue
+	mutantliver = /obj/item/organ/liver
+	mutantstomach = /obj/item/organ/stomach
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human

```

Comment:
vox are people???

Reply:
temporary until i actually code the organs/meat

Reply:
People are secretly vox infiltrators, you can tell because they pocket everything and break into departments to loot